audit/src/main/java/com/ritense/audit/domain/AuditRecord.java

```diff
@@ -53,14 +57,18 @@ public class AuditRecord implements Persistable<AuditRecordId> {
     @Column(name = "created_on", updatable = false)
     private LocalDateTime createdOn = LocalDateTime.now();
 
-    @Type(type = "com.vladmihalcea.hibernate.type.json.JsonType")
+    @Type(value = JsonType.class)
     @Column(name = "audit_event", columnDefinition = "json", updatable = false)
     @JsonView(AuditView.Public.class)
     private AuditEvent auditEvent;
 
     @Column(name = "document_id", updatable = false)
     private UUID documentId;
 
+    @Generated
+    @Column(name = "classname", updatable = false)
+    private String className;
```

Comment:
The `classname` column was already there. The content is generated by the DBMS. But the queries failed on Hibernate 6 because the field was not part of the entity class.